core/trino-spi/src/main/java/io/trino/spi/security/SystemAccessControl.java

```diff
@@ -130,6 +143,22 @@ default void checkCanViewQueryOwnedBy(SystemSecurityContext context, String quer
         denyViewQuery();
     }
 
+    /**
+     * Filter the list of users to those the identity view query owned by the user.  The method
```

Comment:
> Filter the list of users to those the identity view query owned by the user.

@kokosing  What does it mean?

Reply:
The list of users represents here query owners. Method filters that list to only these for which current user will be able to query of theirs. Did it help?